src/main/java/Calculations/GPACalculation.java

```diff
@@ -0,0 +1,167 @@
+package Calculations;
```

Comment:
This class lacks a set of tests. Refer to the src/test folder on how tests are written and implemented and try and write a comprehensive test suite that captures edge cases for each function. This is especially important as this is a math heavy class.